style.css

```diff
@@ -16,6 +20,7 @@ header {
   display: flex;
   justify-content: center;
   background-color: #fff;
+  position: relative;
 }
 
 .top {
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add a position of sticky to your Navbar to make it always visible to the user even when they click on any of the mobile menu options.